v3/heroku.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	Version          = "v3"
+	Version          = ""
```

Comment:
I'm not sure why this changed as this file was generated by Schematic. I'll be glad to restore this if necessary.